policies/release-requirements.md

```diff
@@ -0,0 +1,138 @@
+on in the development branch, the requirements are minimal.
+
+- The CI must pass on the tip of the development branch before the release
+  commits were added to the tree.
+
+Beta Pre-releases
+-----------------
+
+The API and ABI should be stable and the source code should be feature complete
+by the first beta pre-release. The following release requirements apply to beta
+pre-releases.
+
+- The code is functionally complete in regards to the particular release
+  objectives as set by OMC and OTC.
+- There is no remaining refactoring required by OMC or OTC for the release.
+- There are no remaining API changes required for the release.
+  _This applies only to beta releases of a major release as API changes
+  are not allowed on minor releases._
+- There are no remaining API additions required for the release.
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous stable release or previous
+  beta release._
+- All issues or pull requests with the milestone for the beta release
+  are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 3 business days. No changes apart from
+  regression or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- In case of the first beta release the OTC should explicitly approve
+  that the source is ready for a release with a vote.
+
+Major and Minor Releases
+------------------------
+
+As the release comes after the beta releases there is no need to repeat the
+stability requirements as those should be held already by the beta releases.
+
+- All issues [triaged] as regressions on the development branch from which the
+  release is to be done must have a milestone assigned.
+  _Regressions are considered from the previous stable release series._
+- All issues or pull requests with the milestone for the release are closed.
+- There are no outstanding untriaged Coverity issues.
+- Coveralls coverage has not decreased overall from the previous release from
+  the particular development branch.
+- The CI must pass on the tip of the development branch before the release
+  commits are added to the tree, including the daily CI builds.
+- The tree must be frozen for at least 7 days. No changes apart from regression
+  or security fixes should be merged during the freeze.
+- For 2 days before the release there should be no changes to ensure the daily
+  CI builds run on the development tree tip.
+- The OTC should explicitly approve that the source is ready for a release with
+  a vote.
+
+Patch Releases
```

Comment:
It can be noted that we have also call these "release updates", see [policies/stable-release-updates.md](https://github.com/openssl/technical-policies/blob/master/policies/stable-release-updates.md)

Reply:
(if we want to change the nomenclature to "patch releases", we should probably do that across the board...  but still leave a note somewhere saying what we used to call it, to connect the dots.  Frankly, I'd welcome such a change, since "patch release" is a broadly used term)